apps/test/unit/templates/rubrics/AiAssessmentFeedbackTest.jsx

```diff
@@ -4,8 +4,14 @@ import {shallow} from 'enzyme';
 import AiAssessmentFeedback from '@cdo/apps/templates/rubrics/AiAssessmentFeedback';
 
 describe('AiAssessmentFeedback', () => {
+  const mock_aiInfo = {
+    id: 2,
+    learning_goal_id: 2,
+    understanding: 2,
+    ai_confidence: 2,
+  };
   const props = {
-    learningGoalKey: 'learning_key',
+    aiInfo: mock_aiInfo,
```

Comment:
style nit: please avoid underscores in JS variable names when possible. in this case the variable name can just be `mockAiInfo`. we try to only use snake_case when it sneaks through in JSON that comes from the server, and even in those cases the better practice is to have the server do the work to convert any key names in JSON to camelCase.